lib/dowork.js

```diff
@@ -81,29 +81,33 @@ var doWork = {
 	{
```

Comment:
Can't comment since you didn't make a change... but line 48 of this file... missing a return? and two lines after?